docker-compose.yml

```diff
@@ -1,19 +1,21 @@
 version: "3.9"
 services:
   db:
-    image: postgres:latest
+    image: postgres:16
     restart: always
     ports:
       - "5432:5432"
     volumes:
-      - ./tmp/db:/var/lib/postgresql/data
+      - ./tmp/db:/var/lib/postgresql/data:delegated
+      - ./tmp/shared:/tmp/shared:cached
```

Comment:
If `./tmp/{db,shared}` aren't created already, I think these get created and owned by root. That's a bad experience, we don't want people having trouble deleting their tmp folder if they want to.

Reply:
Interesting, this seems to be a linux-only problem. The other side of the coin, for linux's native disk sharing architecture. Ok, I added some instructions  on what seems like the best way to fix this, using  `docker-compose.override.yml`

Reply:
I like using the override fix but it didn't end up fixing it. I now get a file permission error:

`initdb: error: could not access directory "/var/lib/postgresql/data": Permission denied`

Since most of the team isn't running Linux, let's undo the fixes for Linux, mention in the README that the Docker Linux support is buggy and add an issue in tix to document the problems with running docker on Linux.